crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -20,6 +22,21 @@ pub struct DrawState {
 }
 
 impl DrawState {
+    /// Marks the `pipeline` as bound.
+    pub fn set_pipeline(&mut self, pipeline: RenderPipelineId) {
```

Comment:
Why did these get moved? Seems like there's no substantive reason they should have been in a docs PR.

Reply:
To appear in the same order as the fields of `DrawState`. 
Also from a docs perspective, I believe that it makes sense to think about them in that order: set Pipeline, set Bind Groups, set Vertex Buffers, and set Index Buffers.
But I can also revert this, it is not too important IMO.